hooks/post/30-upgrade.rb

```diff
@@ -31,13 +31,15 @@ def import_subscriptions
 end
 
 def elasticsearch_message
-  return true unless Kafo::Helpers.execute('rpm -q elasticsearch')
-
+  if `rpm -q elasticsearch`.gsub!(/\A"|"\Z/, '') != nil
```

Comment:
I still don't see what you mean, there's no `"` in the output of any rpm commands, and regardless of whether the package is installed or not you're going to get nil - so your test isn't really valid, if elasticsearch was installed, it'd skip the steps too.